dapps/templates/boilerplate/package.json

```diff
@@ -34,6 +34,12 @@
   "devDependencies": {
     "embark": "^5.0.0-alpha.8",
     "embark-reset": "^5.0.0-alpha.5",
+    "embarkjs": "^5.0.0-alpha.5",
+    "embarkjs-ens": "^5.0.0-alpha.6",
+    "embarkjs-ipfs": "^5.0.0-alpha.5",
+    "embarkjs-swarm": "^5.0.0-alpha.5",
+    "embarkjs-web3": "^5.0.0-alpha.5",
+    "embarkjs-whisper": "5^.0.0-alpha.5",
```

Comment:
What's up with the mixed number?One is 6, others 5?

Reply:
yep, lerna does that, if nothing changed, the version stays the same.

Reply:
When we do the stable release, everything will bump to 5.0.0 even if there weren't changes. Or that's what it should do — we can double-check when we get to that point.